test/00_operators/OperatorTests.cu

```diff
@@ -167,6 +167,43 @@ TYPED_TEST(OperatorTestsNumericNonComplex, PermuteOp)
   MATX_EXIT_HANDLER();
 }
 
+TYPED_TEST(OperatorTestsNumericNonComplex, ReshapeOp)
+{
+  MATX_ENTER_HANDLER();
+  auto A = make_tensor<TypeParam>({2*4*8*16});
+  for(int i = 0; i < A.Size(0); i++) {
+    A(i) = (TypeParam)i;
+  }
+
+  auto op = reshape<4>(A, {2, 4, 8, 16});
```

Comment:
do we need the template parameter here? can't it be deduced?

Reply:
correct this was left in while debugging.  it can be deduced.